### DISCUSSION TITLE
Does this support mobile use?

### DISCUSSION THREAD

**Discussion by tyler-morales** [quackduck/uniclip#4] [Q&A] [answered]
Is it possible to copy/ paste between laptops and mobile devices?

**Comment by quackduck** [comment, marked as answer]
You can do so using Termux and the Termux API package on Android.
Edit: for complete instructions, please refer to the [README](https://github.com/quackduck/uniclip/blob/master/README.md)

**Reply by tyler-morales** [reply]
@quackduck I got it working on android. Finally something where I can universally copy/ paste. 
Thanks so much!

**Reply by quackduck** [reply]
I'm glad it works! You're welcome!

**Reply by quackduck** [reply]
I'd love feedback. Is it too slow? Is it hard to use? What could be improved?

**Reply by quackduck** [reply]
If you like Uniclip, feel free to give it a star!

**Reply by asahelk** [reply]
@tyler-morales  your Android device is rooted? I got the following message when try to execute 'uniclip':

permission denied

**Comment by quackduck** [comment]
Looking into iPhone support

**Comment by quackduck** [comment]
I need to add documentation for Android :)

**Comment by quackduck** [comment]
@tyler-morales I've added the Android documentation. Feel free to give me feedback!
